nrf5/modules/machine/modmachine.c

```diff
@@ -181,6 +184,9 @@ STATIC const mp_map_elem_t machine_module_globals_table[] = {
 #if MICROPY_PY_MACHINE_PWM
     { MP_OBJ_NEW_QSTR(MP_QSTR_PWM),                 (mp_obj_t)&machine_hard_pwm_type },
 #endif
+#if MICROPY_PY_MACHINE_TEMP
+    { MP_OBJ_NEW_QSTR(MP_QSTR_TEMP),                 (mp_obj_t)&machine_temp_type },
```

Comment:
Ok to rename TEMP to Temp? Class names should be CamelCase, except from abbreviations like PWM, TWI etc.